tests/asset-manager/AaveWeethWithdrawerTest.t.sol

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test, stdStorage, StdStorage} from 'forge-std/Test.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {TransparentProxyFactory} from 'solidity-utils/contracts/transparent-proxy/TransparentProxyFactory.sol';
+
+import {AaveWeethWithdrawer} from '../../src/asset-manager/AaveWeethWithdrawer.sol';
+
+interface IERC721 {
+  function transferFrom(address from, address to, uint256 tokenId) external;
+  function balanceOf(address from) external returns(uint256);
+}
+
+contract AaveWeethWithdrawerTest is Test {
+  using stdStorage for StdStorage;
+  
+  event StartedWithdrawal(uint256[] amounts, uint256 indexed index);
+
+  event FinalizedWithdrawal(uint256 amount, uint256 indexed index);
+  
+  /// At current block withdrawer has a withdrawal NFT
+  ///   this NFT represents an WITHDRAWAL_AMOUNT of EETH that
+  ///   yields FINALIZED_WITHDRAWAL_AMOUNT of ETH when completed.
+  /// Most importantly, this withdrawal is ready to be finalized.
+  uint256 public constant EXISTING_WITHDRAW_REQUESTID = 36881;
+  uint256 public constant WITHDRAWAL_AMOUNT = 9835809070838274;
+  address public constant OWNER = GovernanceV3Ethereum.EXECUTOR_LVL_1;
+  address public constant GUARDIAN = 0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1;
+  address public constant COLLECTOR = address(AaveV3Ethereum.COLLECTOR);
+  IERC20 public constant WETH = IERC20(AaveV3EthereumAssets.WETH_UNDERLYING);
+  IERC20 public constant WEETH = IERC20(AaveV3EthereumAssets.weETH_UNDERLYING);
+  IERC721 public WITHDRAWAL_NFT;
+
+  AaveWeethWithdrawer public withdrawer;
+  
+  /// the collector only has aEthweETH 
+  modifier withWeeth() {
+    vm.prank(0xBdfa7b7893081B35Fb54027489e2Bc7A38275129);
+    WEETH.transfer(COLLECTOR, WITHDRAWAL_AMOUNT);
+    _;
+  }
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 20576941);
+    withdrawer = AaveWeethWithdrawer(payable(0x4D29D2eE3B039c51690e830d8a4ade0d4d03DDbe));
```

Comment:
What is this address?

Reply:
the weeth withdrawer deployed address, as linked in the README